### ISSUE TITLE
The model cannot predict correctly in levenshtein_transformer

### ISSUE THREAD

**Issue by xiaoshengjun** [pytorch/fairseq#2757]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?

Hi，I trained a new model in  levenshtein_transformer.

#### Code

The data is processed by the code below:

python ./fairseq_cli/preprocess.py \
$common_params \
--trainpref $trainpref \
--validpref $validpref \
--testpref $testpref \
--destdir './out/wmt14_be/' \
--workers 60 \
--joined-dictionary \
--thresholdtgt 1 \ 
--thresholdsrc 1 \ 
| tee $OUT/data_bin_$data.log

And the model is trained by:

DATA_BIN=./out/wmt14_be/
CUDA_VISIBLE_DEVICES=$device nohup python train.py $DATA_BIN \
  --save-dir $MODELS \
  --seed 4321 \
  --ddp-backend=no_c10d \
  --task translation_lev \
  --criterion nat_loss \
  --arch levenshtein_transformer \
  --noise random_delete \
  --share-all-embeddings \
  --optimizer adam --adam-betas '(0.9,0.98)' \
  --lr 0.0005 --lr-scheduler inverse_sqrt \
  --min-lr '1e-09' --warmup-updates 10000 \
  --warmup-init-lr '1e-07' --label-smoothing 0.1 \
  --dropout 0.3 --weight-decay 0.01 \
  --decoder-learned-pos \
  --encoder-learned-pos \
  --apply-bert-init \
  --log-format 'simple' --log-interval 100 \
  --fixed-validation-seed 7 \ 
  --max-tokens 8000 \
  --save-interval-updates 10000 \
  --max-epoch 3\
  --max-update 300000   > $OUT/log$exp.out 2>&1 &

and get result is：

CUDA_VISIBLE_DEVICES=$device python ./fairseq_cli/interactive.py ./out/wmt14_be \
    --task translation_lev \
    --path ./out/models20201020_111531/checkpoint_last.pt \
    --iter-decode-max-iter 9 \ 
    --iter-decode-eos-penalty 2 \ 
    --remove-bpe \
    --print-step \
    --batch-size 1 \ 
    --beam 1  

but result is:
2020-10-20 11:26:32 | INFO | fairseq_cli.interactive | Type the input sentence and press return:
Madam President , on a point of order .
S-0     Madam President , on a point of order .
W-0     0.591   seconds
H-0     -2.8191957473754883     <unk>
D-0     -2.8191957473754883     <unk>
P-0     0.0000 -8.4576 0.0000
I find that cause for extremely serious concern .
S-1     I find that cause for extremely serious concern .
W-1     0.070   seconds
H-1     -2.8055810928344727     <unk>
D-1     -2.8055810928344727     <unk>
P-1     0.0000 -8.4167 0.0000

Thank you very much if you can answer my question.

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):0.9
 - PyTorch Version (e.g., 1.0):1.4
 - OS (e.g., Linux):centos
 - How you installed fairseq (`pip`, source):source
 - Build command you used (if compiling from source):pip install --editable ./
 - Python version:3. 7
 - CUDA/cuDNN version:10.1
 - GPU models and configuration:
 - Any other relevant information:
